doc/optparse/option_params.rdoc

```diff
@@ -508,22 +507,3 @@ Executions:
   ["Handler method for -xxx called with value:", true]
   $ ruby method.rb --yyy FOO
   ["Handler method for -yyy called with value:", "FOO"]
-
-=== Terminators
-
-And finally, the terminator parameter <tt>--</tt> tells the options parser
-to ignore any options farther to the right.
-This can be useful if there are options not meant for the current program.
-
-File +terminator.rb+ defines one option <tt>--my_option</tt>.
-
-  :include: ruby/terminator.rb
-
-The first execution fails because <tt>--nosuch</tt> is not a defined option;
-the second succeeds because <tt>--</tt> causes that option to be ignored:
```

Comment:
Removing this section also dropped an inconsistent naming (--nosuch/--other_option).

Reply:
> 
> 
> Removing this section also dropped an inconsistent naming (--nosuch/--other_option).

Not sure what you mean.  Is this drop ok?

Reply:
It is _excellent_ is what I mean.

Reply:
K, thanks!